vue/sbc-common-components/src/services/fee.services.ts

```diff
@@ -21,7 +21,7 @@ export default {
       if (!filing.filingTypeCode) {
         Promise.resolve()
       }
-      const url = `${payApiUrl}fees/${filing.entityType}/${filing.filingTypeCode}?waiveFees=${!!filing.waiveFees}`
+      const url = `${payApiUrl}fees/${filing.entityType}/${filing.filingTypeCode}?waiveFees=${!!filing.waiveFees}&priority=${!!priority}&futureEffective=${!!futureEffective}`
```

Comment:
may be send it only if its true...

Reply:
Thanks @saravankumarpa, I have corrected this, please do re-review